scripts/BilbyReductionScript.py

```diff
@@ -0,0 +1,337 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+# flake8: noqa
+
+import mantid.simpleapi as mantid_api
```

Comment:
To shorten calls to the mantid.simpleapi which is why flake8 might be off, it might make more sense to do:
```python
from mantid.simpleapi import LoadBBY
```